src/serialize.js

```diff
@@ -21,7 +26,10 @@ export default (tree, { search } = {}) =>
             'missedUpdate',
             'hasValue',
             'context',
+            'error',
+            'results',
             ...(search ? [] : ['lastUpdateTime']),
+            ...getNillKeys(x)
```

Comment:
Seems like it would be simpler to use compactObject (or pick by IsNil since I think that also omits false and 0)

Reply:
compact object is immutable 🤔